### PR TITLE
Added card styling and page styling

### PR DESCRIPTION
**Pull request by KirbyDD**

### CODE REVIEW

**Comment by rumizen** [comment]
Love the styling of the cards, nice work!

**Review by rumizen — COMMENTED** [review]
Love the styling on the cards, great work!

File: src/App/App.scss [hunk 1]
```diff
@@ -1,6 +1,12 @@
 .app {
   text-align: center;
-  background-image: url("https://img.wallpapersafari.com/desktop/1920/1080/38/65/Fcx7LA.jpg");
-  background-size: cover;
   height: 970px;
 }
+
+img {
+  position: fixed;
+  left: 0px;
```
Comment (letakeane) [inline src/App/App.scss]: What's going on here?

File: src/CardContainer/CardContainer.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.card-container {
+  display: flex;
```
Comment (letakeane) [inline src/CardContainer/CardContainer.scss]: I like this

File: src/ScrollContainer/ScrollContainer.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.scroll-container {
+  color: white;
```
Comment (letakeane) [inline src/ScrollContainer/ScrollContainer.scss]: Can we make this blanchedalmond instead of white?

**Review by letakeane — COMMENTED** [review]
Overall I like this but I had some questions

File: src/App/App.scss (hunk 1 above)
Comment (letakeane) [reply src/App/App.scss]: Obvviously it's this

**Pull request merged**